core/modules/wbase/Task.h

```diff
@@ -58,7 +58,7 @@ namespace wbase {
 /// Task encapsulates nearly zero logic, aside from:
 /// * constructors
 /// * poison()
-struct Task : public boost::noncopyable {
+struct Task {
```

Comment:
Wait, what happened to the design that needed a change for Task to be copyable? There is/was some code that assumed that a Task wouldn't get copied. Ideally, the assuming code should've been documented (sorry that I forgot). The point may have been to ensure that setPoison/poison calls always get routed to the right Task instance and not another Task instance that looks almost the same.